pyqtgraph/opengl/items/GLTextItem.py

```diff
@@ -0,0 +1,74 @@
+from typing import List, Union
+from OpenGL.GL import *
+from OpenGL.GLUT import *
+import numpy as np
+from pyqtgraph.opengl.GLGraphicsItem import GLGraphicsItem
+
+class GLTextItem(GLGraphicsItem):
+    """Draws text in 3D."""
+
+    pos:Union[np.ndarray, List[float]] = np.array([0.0, 0.0, 0.0])
+    color:np.ndarray = np.array([1.0, 1.0, 1.0, 1.0])
+    text:str = ''
+    font = GLUT_BITMAP_HELVETICA_18
+
+    def __init__(self, **kwds):
+        """All keyword arguments are passed to setData()"""
+        GLGraphicsItem.__init__(self)
+        glopts = kwds.pop('glOptions', 'additive')
+        self.setGLOptions(glopts)
+        self.setData(**kwds)
+        glutInit()
+    
+    def setData(self, **kwds):
+        """
+        Update the data displayed by this item. All arguments are optional;
+        for example it is allowed to update text while leaving colors unchanged, etc.
+
+        ====================  ==================================================
+        **Arguments:**
+        ------------------------------------------------------------------------
+        pos                   (3,) array of floats specifying text location.
+        color                 (4,) array of floats (0.0-1.0) or
+                              tuple of floats specifying
+                              a single color for the entire item.
+        text                  String to display.
+        font                  GLUT font. (Default: GLUT_BITMAP_HELVETICA_18)
+        ====================  ==================================================
+        """
+        args = ['pos', 'color', 'text', 'font']
+        for k in kwds.keys():
+            if k not in args:
+                raise Exception('Invalid keyword argument: %s (allowed arguments are %s)' % (k, str(args)))
+        for arg in args:
+            if arg in kwds:
```

Comment:
probably don't need this if-check given you're checking for this in the for loop above.

Reply:
If this if-check is not done, a KeyError will probably be raised if all the arguments are not specified.
```python
value = kwds[arg]
```

Reply:
oh, right, I misread!